asm/libgcl/desktop.ini

```diff
@@ -0,0 +1,9 @@
+[LocalizedFileNames]

```

Comment:
you can prob delete this file ;)

Reply:
I don't have this file locally

Reply:
its marked as hidden, try show hidden files in windows explorer, maybe desktop.ini is another for .gitignore file